### ISSUE TITLE
Close the "Provide more standardized editable installations" item

### ISSUE THREAD

**Issue by sinoroc** [psf/fundable-packaging-improvements#52]
Should the "*Provide more standardized editable installations*" ([current link](https://github.com/psf/fundable-packaging-improvements/blob/master/FUNDABLES.md#provide-more-standardized-editable-installations)) item be "closed"? It seems to me like it has been implemented in most common packaging tools following [*PEP-660*](https://peps.python.org/pep-0660/).

If yes, should it be removed or moved to a different section? Which one? There is a "*Items that have now been funded*" section, but as far as I can tell it was done without dedicated funding.

**Comment by di** [comment]
Yes, this appears to be mostly done and likely does not require funding anymore.
